packages/contentstack-migration/examples/05-Update-reference-entry-from-mapper.js

```diff
@@ -0,0 +1,138 @@
+const fs = require('fs');
+const chalk = require('chalk');
+module.exports = async ({ migration, stackSDKInstance, managementAPIClient, config }) => {
+  const modules = ['entries', 'assets', 'extensions'];
+
+  const readAllModulesUids = (filePath) => {
+    let uidMapping = [];
```

Comment:
@cs-raj ,It would be preferable to use reduce rather than foreach in this case as you have declared uidMapping as an array and are expecting an object at the end.

Reply:
It's typing error, the uidMapping will be declared as an Object only